raphtory/src/core/mod.rs

```diff
@@ -17,15 +17,13 @@
 //!
 //! `raphtory` supports  support for the following platforms:
 //!
-//! **Note** they must have Rust 1.53 or later.
+//! **Note** they must have Rust 1.80 or later.
```

Comment:
Makes me wonder if there is a way to get this from the Cargo.toml automatically? This is bound to be out of date again next time we update it.